README.md

```diff
@@ -8,9 +8,65 @@ _(Coming soon)_
 ## Documentation
 _(Coming soon)_
 
+## Building Native Android WebRTC
+**WebRTC Android can only be built using Linux, Recommended OS is Ubuntu**
+
+### Setup Depot Tools
+
+```bash
+git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
+
+# Preferrably but the below in your shell file
+export PATH=$PATH:/path/to/depot_tools
+```
+
+### Install Build Dependencies (Ubuntu)
+
+```bash
+cd /path/to/depot_tools
+./build/install-build-deps.sh
+```
+
+### Build Native WebRTC Libraries
+
+```bash
+mkdir webrtc-android-checkout
+cd webrtc-android-checkout
+fetch --nohooks webrtc_android
+gclient sync
+cd src
+
+# Checkout branch m74
+git checkout -b m74 refs/remotes/branch-heads/m74
+gclient sync
+
+# Accept Android SDK Licences
+yes | ./third_party/android_tools/sdk/tools/bin/sdkmanager --licenses
+
+# Apply the path located below
+
+# Build Android WebRTC
+gn gen out/android-webrtc-m74 --args='target_os = "android" target_cpu = "x86" is_debug = false rtc_include_tests = false rtc_use_h264 = true proprietary_codecs = true ffmpeg_branding = "Chrome" use_openh264 = true is_component_build = false is_clang = true rtc_enable_protobuf = false use_rtti = true use_custom_libcxx = false'
```

Comment:
Recommend to use `./tools_webrtc/android/build_aar.py` to generate libs for all supported arch.